py{310,311,312}-djmain tests.test_commands.CreateApplicationTest failing

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->

djmain has changed the wording of a user not found error message, causing the test for a failed validation to fail:

```
______________________________________________ CreateApplicationTest.test_validation_failed_message _______________________________________________

self = <tests.test_commands.CreateApplicationTest testMethod=test_validation_failed_message>

    def test_validation_failed_message(self):
        output = StringIO()
        call_command(
            "createapplication",
            "confidential",
            "authorization-code",
            "--redirect-uris=http://example.com http://example2.com",
            "--user=783",
            stdout=output,
        )
    
        self.assertIn("user", output.getvalue())
        self.assertIn("783", output.getvalue())
>       self.assertIn("does not exist", output.getvalue())
E       AssertionError: 'does not exist' not found in "\x1b[31;1mPlease correct the following errors:\n - user: ['user instance with id 783 is not a valid choice.']\x1b[0m\n"

tests/test_commands.py:145: AssertionError
```

**To Reproduce**
<!-- Steps to reproduce the behavior -->
```
tox -e py310-djmain
```

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->
All tests should have passed

**Version**
<!-- Version of django-oauth-toolkit -->

<!-- Have you tested with the latest version and/or master branch? -->
<!-- Replace '[ ]' with '[x]' to indicate that. -->
- [ ] I have tested with the latest published release and it's still a problem.
- [x] I have tested with the master branch and it's still a problem.

**Additional context**
<!-- Add any other context about the problem here. -->

--- a/tests/test_commands.py
+++ b/tests/test_commands.py
@@ -145,6 +145,5 @@ class CreateApplicationTest(TestCase):
         self.assertIn("783", output_str)
         # newer Django (>5.1) changes the error message from "does not exist" to "is not a valid choice"
         self.assertTrue(
-            any(substring in output_str for substring in ["does not exist", "is not a valid choice"]),
-            f"Output did not contain 'does not exist' or 'is not a valid choice'. Actual output: {output_str}",
+            any(substring in output_str for substring in ["does not exist", "is not a valid choice"])
         )

--- a/tests/test_commands.py
+++ b/tests/test_commands.py
@@ -140,6 +140,11 @@ class CreateApplicationTest(TestCase):
             stdout=output,
         )
 
-        self.assertIn("user", output.getvalue())
-        self.assertIn("783", output.getvalue())
-        self.assertIn("does not exist", output.getvalue())
+        output_str = output.getvalue()
+        self.assertIn("user", output_str)
+        self.assertIn("783", output_str)
+        # newer Django (>5.1) changes the error message from "does not exist" to "is not a valid choice"
+        self.assertTrue(
+            any(substring in output_str for substring in ["does not exist", "is not a valid choice"]),
+            f"Output did not contain 'does not exist' or 'is not a valid choice'. Actual output: {output_str}",
+        )

--- a/tests/test_commands.py
+++ b/tests/test_commands.py
@@ -130,6 +130,8 @@ class CreateApplicationTest(TestCase):
         self.assertEqual(app.algorithm, "RS256")
 
     def test_validation_failed_message(self):
+        import django
+
         output = StringIO()
         call_command(
             "createapplication",
@@ -143,7 +145,7 @@ class CreateApplicationTest(TestCase):
         output_str = output.getvalue()
         self.assertIn("user", output_str)
         self.assertIn("783", output_str)
-        # newer Django (>5.1) changes the error message from "does not exist" to "is not a valid choice"
-        self.assertTrue(
-            any(substring in output_str for substring in ["does not exist", "is not a valid choice"])
-        )
+        if django.VERSION < (5, 2):
+            self.assertIn("does not exist", output_str)
+        else:
+            self.assertIn("is not a valid choice", output_str)

